tests/test_kem.c

```diff
@@ -31,6 +38,89 @@ static void OQS_print_hex_string(const char *label, const uint8_t *str, size_t l
 	printf("\n");
 }
 
+/* mlkem rejection key testcase */
```

Comment:
OK -- now I get it. But then this should all be #ifdef'd on that algorithm family (being present/enabled), no?

Reply:
Testcase under mlkem conditional check now